openreview/conference/templates/programchairWebfield.js

```diff
@@ -717,13 +718,23 @@ var buildSeniorAreaChairGroupMaps = function(noteNumbers, seniorAreaChairGroups)
   };
 };
 
-var getOfficialReviews = function(notes) {
+var getOfficialReviews = function (notes, anonReviewerGroups, submissionNumber) {
   var ratingExp = /^(\d+): .*/;
 
   var reviewByAnonId = {};
 
   _.forEach(notes, function(n) {
-    var anonId = getNumberfromGroup(n.signatures[0], ANON_REVIEWER_NAME);
+    var anonId;
+    if (IS_NON_ANONYMOUS_VENUE) {
+      var reviewerProfileId = n.signatures[0];
+      anonId = getNumberfromGroup(
+        (anonReviewerGroups && anonReviewerGroups[submissionNumber] && anonReviewerGroups[submissionNumber][reviewerProfileId]) || '',
```

Comment:
can you pass `anonReviewerGroups[submissionNumber]` as a parameter directly? so you don't need to pass two variables?